src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -54,71 +61,115 @@ interface EncryptionUserSettingsTabProps {
     initialState?: State;
 }
 
+const useKeyBackupIsEnabled = (): boolean | undefined => {
+    const [isEnabled, setIsEnabled] = useState<boolean | undefined>(undefined);
+    const [loading, setLoading] = useState(true);
+
+    const matrixClient = useMatrixClientContext();
+
+    const checkStatus = useCallback(async () => {
+        const crypto = matrixClient.getCrypto()!;
+        const info = await crypto.getKeyBackupInfo();
+        setIsEnabled(Boolean(info?.version));
+    }, [matrixClient, setIsEnabled]);
+
+    useEffect(() => {
+        (async () => {
+            await checkStatus();
+            setLoading(false);
+        })();
+    }, [checkStatus, setLoading]);
+
+    useEventEmitter(matrixClient, ClientEvent.AccountData, (event: MatrixEvent): void => {
```

Comment:
`useTypedEventEmitter` doesn't work here? (genuine question)

Reply:
Ah no, I just forgot it!